### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by Fishbowler**
Make htmlproofer happy by fixing broken links

### CODE REVIEW

**Comment by Fishbowler** [comment]
Hmmmm. Just fixing the html-proofer doesn't fix the build. Gotta fix the linting too. On it..

File: SUMMARY.md [hunk 1]
```diff
@@ -2,563 +2,563 @@
```
Comment (MartinSchoeler) [inline SUMMARY.md]: This file is generated by gitbook, so I think any changes done here will be reverted by it

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]
I think only one file needs to be changed

File: SUMMARY.md (hunk 1 above)
Comment (Fishbowler) [reply SUMMARY.md]: I've modified the commit and removed this file, rebased, force pushed.

**Comment by Fishbowler** [comment]
I think this is all fixed up?

**Review by MartinSchoeler — APPROVED** [review]
Looks good to me! Thanks for helping out!

**Pull request merged**